### PR TITLE
Fix the links to the wiki

### PR DESCRIPTION
**Pull request by Prometheos2**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -33,8 +33,8 @@ For more details, [see the official docs](https://developer.mozilla.org/Add-ons/
 
 * Text assets to translate: [`assets/texts`](assets/texts)
 * Screenshots: [`assets/screenshots`](assets/screenshots)
-* Wiki to translate: [wiki](/wiki)
-* Sidebar file for adding language: [`_Sidebar` file](/wiki/_Sidebar/_edit)
+* Wiki to translate: [wiki](https://github.com/rugk/awesome-emoji-picker/wiki)
```
Comment (rugk) [inline CONTRIBUTING.md]:
```suggestion
* Wiki to translate: [wiki](../wiki)
```

should be better as it is relative, is not it?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -33,8 +33,8 @@ For more details, [see the official docs](https://developer.mozilla.org/Add-ons/
...
+* Sidebar file for adding language: [`_Sidebar` file](https://github.com/rugk/awesome-emoji-picker/wiki/_Sidebar/_edit)
```
Comment (rugk) [inline CONTRIBUTING.md]:
```suggestion
* Sidebar file for adding language: [`_Sidebar` file](../wiki/_Sidebar/_edit)
```

same

File: CONTRIBUTING.md (hunk 1 above)
Comment (Prometheos2) [reply CONTRIBUTING.md]: We come from `blob/master/CONTRIBUTING.md`, so it would be `../../`, but good point

**Comment by rugk** [comment]
Thanks! Merged and tested the live version, seems to work. :smile: 

Now, again, I'd ask if you could also adjust that for the other add-ons, if you want. :smile: 
And if you wanna be extra nice ( :wink: ), could you maybe also create a PR for my [AddonTemplate](https://github.com/TinyWebEx/AddonTemplate)? (also for the previous typo fixes) :blush:

**Pull request merged**